src/AutoRest.CSharp/Mgmt/Decorator/Transformer/FrameworkTypeUpdater.cs

```diff
@@ -42,7 +42,7 @@ public static void ValidateAndUpdate(CodeModel codeModel)
                         }
                         else
                         {
-                            throw new InvalidOperationException($"{objSchema.Name} has a property named \"Type\" which is not allowed. Please use \"x-ms-client-name\" to rename the property for the client.");
+                            throw new InvalidOperationException($"{objSchema.Name} has a property named \"{property.CSharpName()}\" which is not allowed. Please add \"{objSchema.Name}.{property.Language.Default.Name}: YourNewPropertyName\" into the `rename-mapping` section to rename the property for the client.");
```

Comment:
shall we specify this change is in `autorest.md` as well?

Reply:
better to output `{YourNewPropertyName}`, so I guess the string should be `{{YourNewPropertyName}}`

Reply:
Fixed